modules/installation/proc-config-rhdh-custom-resource.adoc

```diff
@@ -1,4 +1,53 @@
 [id="proc-config-rhdh-custom-resource_{context}"]
 = Configuring the {product-short} Custom Resource
 
-Updates to the Backstage custom resource (CR) are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
\ No newline at end of file
+You can change the configuration of the Custom Resource (CR) to program the way that the Operator manages your {product} instance.
+
+[NOTE]
+====
+Updates to the Backstage CR are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
+====
+
+.Prerequisites
+* You have an administrative role in {ocp-brand-name}
+* You have added a custom application configuration file to your {ocp-short} instance
+
+.Procedure
+
+To configure your {product-short} CR file for your {product-short} instance, complete the following steps:
+
+. Add a custom application configuration file to {ocp-short} and reference it in the CR. In {ocp-short}, you can use the following example as a base template to create a ConfigMap, such as `app-config-rhdh.yaml`:
++
+[source,yaml,subs="attributes+"]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-config-rhdh
+data:
+  "app-config-rhdh.yaml": |
+    app:
+      title: {product}
+      baseUrl: https://backstage-developer-hub-my-ns.apps.ci-ln-vtkzr22-72292.origin-ci-int-gce.dev.rhcloud.com
```

Comment:
```suggestion
      baseUrl: https://backstage-<CUSTOM_RESOURCE_NAME>-<NAMESPACE_NAME>.<OPENSHIFT_INGRESS_DOMAIN>
```

The whole YAML is a template, right? So maybe we should not hardcode values there..